guide/14-deep-learning/dl-on-time-series-data.ipynb

```diff
@@ -14,14 +14,17 @@
     "# Table Of Contents\n",
```

Comment:
<p>remove line: The next step is initializing the timeseries model for model training.</p><p> </p><h1>TimeSeriesModel </h1><p>TimeSeriesModel in&nbsp;<code>arcgis.learn</code>&nbsp;is built on top of&nbsp;??&nbsp;library. The model training and inferencing workflow are similar to computer vision models in&nbsp;<code>arcgis.learn</code>.</p><p>Run the command below to see what backbones are supported for the timeseries task. </p><p> </p><pre class="ql-syntax" spellcheck="false">print(TimeSeriesModel.supported_backbones)
</pre><p> </p><p>also add imports in a call</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1033/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
<p>line removed, info on baackbones/model architecture is already there, Timeseries.supported_backbones gives error - Timeseries has no attribute supported_backbones</p>